Frameworks/Foundation/NSException.mm

```diff
@@ -216,7 +244,18 @@ - (HRESULT)_hresult {
         return [hresultValue unsignedIntValue];
     }
 
-    TraceWarning(L"NSException", L"Called -hresult on an NSException without an HRESULT!");
-    return E_UNEXPECTED;
+    return [[self class] _HRESULTForExceptionName:self.name];
+}
+
+- (void)_processException {
+    HRESULT hr = [self _hresult];
```

Comment:
Why? Do we normally check to this?  I don't see us doing this consistently in other files.

---
In reply to: [93125224](https://github.com/Microsoft/WinObjC/pull/1570#discussion_r93125224) [](ancestors = 93125224)